app/jobs/s3_east_delivery_job.rb

```diff
@@ -1,10 +1,12 @@
 # frozen_string_literal: true
 
-# Same as parent class, just a different queue.
+# @see PreservationCatalog::Aws for how S3 credentials and bucket are configured
 # @note This name is slightly misleading, as this class solely deals with AWS US East 1 endpoint
-class S3EastDeliveryJob < S3WestDeliveryJob
+class S3EastDeliveryJob < AbstractDeliveryJob
```

Comment:
How about S3AWSEastDeliveryJob?  Or even `AWSEastDeliveryJob` to make it more like IBM?

Reply:
See above.